python/sglang/utils.py

```diff
@@ -300,27 +301,25 @@ def download_and_cache_file(url: str, filename: Optional[str] = None):
     return filename
 
 
-def execute_shell_command(command: str) -> subprocess.Popen:
+def lauch_sglang_server(command: str) -> subprocess.Popen:
```

Comment:
call it `execute_shell_command`

BTW `lauch_sglang_server` should be `launch_sglang_server`